org.standardnotes.standardnotes.yml

```diff
@@ -55,18 +55,18 @@ modules:
         $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror'
 
       # Translated from npm run setup
-      - yarn --offline
-      # - yarn --offline --pure-lockfile --cwd ./web
-      - yarn --offline --cwd ./app
+      - ln -s ~/.node-gyp $XDG_CACHE_HOME/node-gyp; yarn --offline --pure-lockfile 
```

Comment:
You've probably mixed up source and target here. `~/.node-gyp` should point to `$XDG_CACHE_HOME/node-gyp`, not the other way around.

Reply:
Well, the other way around doesn't work either. Fixed by `npm_config_nodedir: /run/build/standardnotes/flatpak-node/cache/node-gyp/9.4.2` anyway